### PR TITLE
Feature: catagory management

### PR DESCRIPTION
**Pull request by rizwans-cuelogic**
added category catagory management for admin . 
file change:admin..py  model.py

### CODE REVIEW

File: project4/project/ebs/admin.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    ordering = ['username']
+    actions = [mark_inactive, mark_active]
+
+    def display_orgname(self, request):
+        if Organisation.objects.filter(user__id=request.id):
+            return 'Client'
+    display_orgname.short_description = 'Role'
+
+
+class CatagoriesModelAdmin(admin.ModelAdmin):
```
Comment (ananthu-nair707) [inline project4/project/ebs/admin.py]: Change "Catagory" to "Category" everywhere.

File: project4/project/ebs/models.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            text_content = strip_tags(html_content)
+            from_email = settings.EMAIL_HOST_USER
+            msg = EmailMultiAlternatives(
+                subject, text_content, from_email, [instance.email])
+            msg.attach_alternative(html_content, "text/html")
+            msg.send()
+        else:
+            pass
+    except:
+        print "can not send email"
```
Comment (ananthu-nair707) [inline project4/project/ebs/models.py]: Remove print statement here. In order to log a particular error use django's logging functionality.
https://docs.djangoproject.com/en/1.11/topics/logging/

File: project4/project/ebs/models.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+class Blog(models.Model):
+    organisation=models.ForeignKey(Organisation,blank=True,on_delete=models.CASCADE)
+    title=models.CharField(max_length=125)
+    description=models.TextField()
+    tags = models.CharField(max_length=125,blank=True,null=True)
+    updated=models.DateTimeField(auto_now=True,auto_now_add=False)
+    timestamp=models.DateTimeField(auto_now=False,auto_now_add=True)
+    published=models.DateTimeField()
+    publishedstate=models.BooleanField(default=False)
+    Draft=models.BooleanField(default=False)
```
Comment (ananthu-nair707) [inline project4/project/ebs/models.py]: keep variable names small case.

File: project4/project/ebs/models.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    name=models.CharField(max_length=125)
+    state=models.BooleanField(default=True)
+
+    def __unicode__(self):
+        return self.name
+
+class Blog(models.Model):
+    organisation=models.ForeignKey(Organisation,blank=True,on_delete=models.CASCADE)
+    title=models.CharField(max_length=125)
+    description=models.TextField()
```
Comment (ananthu-nair707) [inline project4/project/ebs/models.py:67]: Is description nullable?

File: project4/project/ebs/models.py [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+
+class forgotpassword(models.Model):
+    username = models.ForeignKey(User, on_delete=models.CASCADE)
+    activation_key = models.CharField(max_length=50)
+    link_time = models.DateTimeField(default=datetime.now, blank=True)
+
+
+class Catagories(models.Model):
+    name=models.CharField(max_length=125)
```
Comment (ananthu-nair707) [inline project4/project/ebs/models.py]: Is name nullable?

File: project4/project/ebs/models.py [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+        return self.name
+
+class Blog(models.Model):
+    organisation=models.ForeignKey(Organisation,blank=True,on_delete=models.CASCADE)
+    title=models.CharField(max_length=125)
+    description=models.TextField()
+    tags = models.CharField(max_length=125,blank=True,null=True)
+    updated=models.DateTimeField(auto_now=True,auto_now_add=False)
+    timestamp=models.DateTimeField(auto_now=False,auto_now_add=True)
+    published=models.DateTimeField()
```
Comment (ananthu-nair707) [inline project4/project/ebs/models.py]: Set default published date as datetime.now.

**Review by ananthu-nair707 — CHANGES_REQUESTED** [review]

File: project4/project/ebs/models.py (hunk 4 above)
Comment (rizwans-cuelogic) [reply project4/project/ebs/models.py:67]: description is required

Comment (rizwans-cuelogic) [reply project4/project/ebs/models.py:67]: not nullable

File: project4/project/ebs/models.py [hunk 7]
```diff
@@ -10,8 +10,8 @@
 from django.template.loader import render_to_string, get_template
 from django.utils.html import strip_tags
 from django.core.mail import EmailMultiAlternatives
-
-
+from django.utils import timezone
+import logging
```
Comment (ananthu-nair707) [inline project4/project/ebs/models.py]: Maintain sequencing of imports.

File: project4/project/ebs/models.py [hunk 8]
```diff
@@ -51,3 +51,38 @@ class forgotpassword(models.Model):
...
+
+class Comment(models.Model):
+    text=models.CharField(max_length=225)
+    blog=models.ForeignKey(Blog,blank=True,on_delete=models.CASCADE)
+
+class BlogFile(models.Model):
+    attachments=models.FileField(upload_to='blogimages/')
+    blog = models.ForeignKey(Blog,blank=True,on_delete=models.CASCADE)
+
+
```
Comment (ananthu-nair707) [inline project4/project/ebs/models.py]: EOF fix here.

**Review by ananthu-nair707 — CHANGES_REQUESTED** [review]
Please make final changes.

**Review by ananthu-nair707 — APPROVED** [review]

**Pull request merged**